### DISCUSSION TITLE
sgconcog.yml options is reletive path? or absolute path?

### DISCUSSION THREAD

**Discussion by zoomer-k** [ast-grep/ast-grep#1215] [Q&A]
https://ast-grep.github.io/reference/sgconfig.html

I readed this page 
ruleDirs seems reletive

but librarypath is not clearly.

https://ast-grep.github.io/advanced/custom-language.html

this page says absolute path could use

**Comment by HerringtonDarkholme** [comment]
It should support both and you can give it a try.

**Reply by zoomer-k** [reply]
thankyou for your answer.

I can do it.

yaml have some trick

1. when use "" for path,  backslash need escape \\ but colon is not need
2. dont need to use ""
```
libraryPath: C:\test\test.dll
```
or
```
libraryPath: "C:\\test\\test.dll"
```
https://runebook.dev/en/docs/ansible/user_guide/windows_usage#path-formatting-for-windows
